tests/test_verify.py

```diff
@@ -0,0 +1,57 @@
+""" Test the verification code """
+import unittest.mock as mock
+
+import django
+from geospaas.catalog.models import Dataset, DatasetURI
+
+import geospaas_harvesting.verify as verify
+from tests.test_ingesters import IngesterTestCase as itc
```

Comment:
You don't need to use an alias since there is no other class named "IngesterTestCase" here.

Reply:
Done in the next commit.